lib/backends/webgl/base-webgl-context.ts

```diff
@@ -126,12 +126,12 @@ export abstract class BaseWebGLContext implements WebGLContext, Disposable {
   }
   readTexture(
       texture: WebGLTexture, width: number, height: number, dataSize: number, dataType: Encoder.DataType,
-      channels: number): Encoder.DataArrayType {
+      channels: number, forceRgbaReads?: boolean): Encoder.DataArrayType {
```

Comment:
should we access WebGLOptions from the backend object, instead of passing all those flags from functions to functions